charts/spire-client-example/README.md

```diff
@@ -1,15 +1,18 @@
 # spire-client-example
 
+
```

Comment:
Remove the changes on this Chart from this PR.

Reply:
This change was made by helm-docs.sh script. I will revert it.